README.md

```diff
@@ -0,0 +1,21 @@
+# Generic connects
+
+This project contains extensions for base "[Connect](https://github.com/th2-net/th2-conn)" component
+to support communication with the target system using common protocols.
+
+Currently, it contains extensions for following protocols:
++ FIX
+
+## Build connects
+
+To build the "Connect" component you will need a Docker installed on your machine.
```

Comment:
To build the "Connect" component you will need **to have** Docker installed on your machine.